packages/os/autoscaling-warm-pool.service

```
@@ -0,0 +1,17 @@
+[Unit]
+Description=Run tasks reliant on IMDS
+After=network-online.target settings-applier.service
+Requires=settings-applier.service
+Wanted=network-online.target
```

These dependencies can just be:
```suggestion
After=configured.target
Wants=configured.target
```

---

I think It should be 
```
After=network-online.target configured.target
Wants=network-online.target configured.target
```

`warm-pool-wait.service` needs wait `network` to be ready and then be able to fetch instance.

---

`configured.target` depends on `preconfigured.target` which requires many services that depend on`network-online.target`, so you can safely do what @bcressey suggested :+1:

---

We discuss offline, and we think we still need add `network-online.target` here.